R/froll.R

```diff
@@ -34,13 +34,7 @@ partial2adaptive = function(x, n, align) {
   }
 }
 
-froll = function(fun, x, n, fill=NA, algo=c("fast","exact"), align=c("right","left","center"), na.rm=FALSE, has.nf=NA, adaptive=FALSE, partial=FALSE, hasNA) {
-  stopifnot(!missing(fun), is.character(fun), length(fun)==1L, !is.na(fun))
-  if (!missing(hasNA)) {
-    warningf("hasNA is deprecated, use has.nf instead")
-    if (missing(has.nf)) has.nf = hasNA
-  } # remove check on next major release
-  algo = match.arg(algo)
+froll = function(fun, x, n, fill=NA, algo, align=c("right","left","center"), na.rm=FALSE, has.nf=NA, adaptive=FALSE, partial=FALSE, FUN, rho) {
```

Comment:
rho is somehow missing from the .Rd files

Reply:
It is not in API, if there would be missing arg in Rd the  package check would fail.
This arg is used for frollapply, and will be removed in next PR which rewrites frollapply.